ClientApp/viking-vault/src/ExchangeResponseModal.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import './ExchangeResponseModal.css'
+
+interface IExchangeResponseModalProps{
+    open: boolean;
+    closeModal: any;
+    message : string;
+}
+
+class ExchangeResponseModal extends React.Component<IExchangeResponseModalProps, any>{ 
+    constructor(props : IExchangeResponseModalProps){
+        super(props);
+    }
```

Comment:
The constructor is optional, in this case you don't need it

Reply:
Deleted it.